<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fm_gestaoestoque</class>
 <widget class="QDialog" name="fm_gestaoestoque">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>570</width>
    <height>478</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_13">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab_novoproduto">
      <attribute name="title">
       <string>novo produto</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_12">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Codigo do produto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_codigo"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Descrição do produto</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_descricao"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Qtde estoque</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_estoque"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Fornecedor</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="txt_fornecedor"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Valor de compra</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_compra"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Valor de venda</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="txt_venda"/>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="btn_novoproduto">
         <property name="text">
          <string>Novo</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_gravar">
         <property name="text">
          <string>Gravar</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_gestaoproduto">
      <attribute name="title">
       <string>Gestão produto</string>
      </attribute>
      <widget class="QPushButton" name="btn_ge_gravar">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>350</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Gravar</string>
       </property>
      </widget>
      <widget class="QPushButton" name="btn_ge_excluir">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>350</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Excluir</string>
       </property>
      </widget>
      <widget class="QTableWidget" name="tw_ge_produtos">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>0</y>
         <width>191</width>
         <height>231</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>240</y>
         <width>201</width>
         <height>111</height>
        </rect>
       </property>
       <property name="title">
        <string>Filtrar por:</string>
       </property>
       <widget class="QRadioButton" name="rb_ge_codigo">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>131</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Codigo do produto</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
       <widget class="QRadioButton" name="rb_ge_produto">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Descrição do produto</string>
        </property>
       </widget>
      </widget>
      <widget class="QPushButton" name="btn_ge_filtrar">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>360</y>
         <width>75</width>
         <height>24</height>
        </rect>
       </property>
       <property name="text">
        <string>Pesquisar</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="txt_ge_filtro">
       <property name="geometry">
        <rect>
         <x>350</x>
         <y>360</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>261</width>
         <height>301</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_14">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Codigo do produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_codigo"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Produto</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_produto"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Quantidade</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_qtd"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Valor compra</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_compra"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Valor venda</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_venda"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Fornecedor</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txt_ge_fornecedor"/>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
